internal/kafka/internal/services/quota/internal/utils/misc.go

```diff
@@ -0,0 +1,40 @@
+package utils
```

Comment:
love the file name `misc.go` in this package ;-). I cannot think of a better name to suggest but I find the filename not clear about the utility functions that are being offered. 
If the utility functions are not common enough for us to come with a common and clear name then an alternative would be to have a utility file per function.